lib/DueDatePicker/styles.scss

```diff
@@ -0,0 +1,349 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$duedate-popover-width: 304px !default;
+$duedate-timepicker-width: 80px !default;
+$duedate-timepicker-height: 220px !default;
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.duedate__popover.duedate__popover {
+  max-width: $duedate-popover-width;
+  width: $duedate-popover-width;
+  margin-top: $layout-spacing-base / -2;
+
+  .arrow {
```

Comment:
Is there a task to hide all arrows for popovers? If so shall we do this globally? 😄

Pinging @jamesdarracott & @nhqe for input.